pkg/health/client/modules.go

```diff
@@ -32,17 +42,16 @@ func GetAndFormatModulesHealth(w io.Writer, clt ModulesHealth, verbose bool) {
 		return
 	}
 	if verbose {
-		fmt.Fprintln(w)
+		r := newRoot(rootNode)
 		for _, m := range resp.Payload.Modules {
-			n := &cell.StatusNode{}
-			if err := json.Unmarshal([]byte(m.Message), n); err != nil {
-				panic(err)
-			}
 			if m.Level == string(cell.StatusUnknown) {
 				continue
 			}
-			fmt.Fprintf(w, "%s", n.StringIndent(2))
+			if err := buildTree(r, m.Message); err != nil {
+				fmt.Fprintf(w, "Modules Health rendering failed: %s\n", err)
```

Comment:
Should we do `os.Exit(1)` here?

Reply:
We may not be end of line here? imho this is ok.

Reply:
Makes sense, this failing would certainly be unexpected but we can try to keep plowing forward.